tc/tc_util.c

```diff
@@ -192,6 +192,86 @@ static const struct rate_suffix {
 };
 
 
+int get_percent_rate(unsigned int *rate, const char *str, char *dev)
+{
+	long value;
```

Comment:
Rename `value` to be whatever unit we are reading for the speed (maybe `max_rate_bytes` or `max_rate_mbytes`, whatever the unit is we're reading here).